src/colvarscript_commands.h

```diff
@@ -0,0 +1,402 @@
+// -*- c++ -*-
+
+#ifndef COLVARSCRIPT_COMMANDS_H
```

Comment:
Copying over this file from `colvarscript-files` and adding to it the new implementations of the older scripting commands may work. However, the corresponding commands would have to be deleted from `colvarscript.h` as well: at the moment they seem to be duplicated.